sys/android-architecture.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+	echo "Please enter an android architecture..."
+fi
+
+
+case "$1" in
+	aarch64)
```

Comment:
indent the cases at the same level of the case. like we do in C following our guidelines